packages/ui-components/src/components/tooltip/tooltip.tsx

```diff
@@ -28,6 +28,8 @@ export class KvTooltip implements ITooltip {
 	@Prop({ reflect: true }) disabled?: boolean = false;
 	/** @inheritdoc */
 	@Prop({ reflect: false }) contentElement?: HTMLElement = null;
+	/** @inheritdoc */
+	@Prop({ reflect: false }) display?: boolean = false;
```

Comment:
Despite this being a good way to handle it, I have a different suggestion, which IMO is cleaner. I think you should have a prop called 'behaviour' that would be either 'click', 'hover' or 'truncate'. Then, the tooltip will only be shown if either one of the following conditions is met:
- Behaviour is 'click' and the tooltip content is clicked.
- Behaviour is 'hover' and the tooltip content is with the mouse over.
- Behaviour is 'truncate', the tooltip content width is not enough and is truncated and it is with the mouse over.

What do you think?

Reply:
Yeap, I didn't particularly appreciate how I did this. That's an excellent suggestion I like it a lot  💪

Reply:
I have some doubts about click behaviour. What we intent with it? Only open the tooltip if a click is done in the element? Or it's only to have the cursor point in the element?

Reply:
Be careful, we already have a tooltip for click behavior with name kv-toggle-tip. Please speak with @GoncaloCamaz before startiing.